packages/foreman/nodejs-patternfly/nodejs-patternfly.spec

```diff
@@ -0,0 +1,93 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name patternfly
+
+Name: %{?scl_prefix}nodejs-patternfly
+Version: 3.59.5
+Release: 1%{?dist}
+Summary: This reference implementation of PatternFly is based on [Bootstrap v3](http://getbootstrap
+License: MIT
+Group: Development/Libraries
+URL: https://github.com/patternfly/patternfly-3
+Source0: https://registry.npmjs.org/patternfly/-/patternfly-%{version}.tgz
+%if 0%{?!scl:1}
+BuildRequires: nodejs-packaging
+%endif
+Requires: npm(@types/c3) >= 0.6.0
+Requires: npm(@types/c3) < 0.7.0
+Requires: npm(bootstrap) >= 3.4.1
+Requires: npm(bootstrap) < 3.5.0
+Requires: npm(bootstrap-datepicker) >= 1.7.1
+Requires: npm(bootstrap-datepicker) < 2.0.0
+Requires: npm(bootstrap-sass) >= 3.4.0
+Requires: npm(bootstrap-sass) < 4.0.0
+Requires: npm(bootstrap-select) = 1.12.2
+Requires: npm(bootstrap-slider) >= 9.9.0
+Requires: npm(bootstrap-slider) < 10.0.0
+Requires: npm(bootstrap-switch) = 3.3.4
+Requires: npm(bootstrap-touchspin) >= 3.1.1
+Requires: npm(bootstrap-touchspin) < 3.2.0
+Requires: npm(c3) >= 0.4.11
+Requires: npm(c3) < 0.5.0
+Requires: npm(d3) >= 3.5.17
+Requires: npm(d3) < 3.6.0
+Requires: npm(datatables.net) >= 1.10.15
+Requires: npm(datatables.net) < 2.0.0
+Requires: npm(datatables.net-colreorder) >= 1.4.1
+Requires: npm(datatables.net-colreorder) < 2.0.0
+Requires: npm(datatables.net-colreorder-bs) >= 1.3.2
+Requires: npm(datatables.net-colreorder-bs) < 1.4.0
+Requires: npm(datatables.net-select) >= 1.2.0
+Requires: npm(datatables.net-select) < 1.3.0
+Requires: npm(drmonty-datatables-colvis) >= 1.1.2
+Requires: npm(drmonty-datatables-colvis) < 1.2.0
+Requires: npm(eonasdan-bootstrap-datetimepicker) >= 4.17.47
+Requires: npm(eonasdan-bootstrap-datetimepicker) < 5.0.0
+Requires: npm(font-awesome) >= 4.7.0
+Requires: npm(font-awesome) < 5.0.0
+Requires: npm(font-awesome-sass) >= 4.7.0
+Requires: npm(font-awesome-sass) < 5.0.0
+Requires: npm(google-code-prettify) >= 1.0.5
+Requires: npm(google-code-prettify) < 1.1.0
+Requires: npm(jquery) >= 3.4.1
+Requires: npm(jquery) < 3.5.0
+Requires: npm(jquery-match-height) >= 0.7.2
+Requires: npm(jquery-match-height) < 0.8.0
+Requires: npm(moment) >= 2.19.1
+Requires: npm(moment) < 3.0.0
+Requires: npm(moment-timezone) >= 0.4.1
+Requires: npm(moment-timezone) < 0.5.0
+Requires: npm(patternfly-bootstrap-combobox) >= 1.1.7
+Requires: npm(patternfly-bootstrap-combobox) < 1.2.0
+Requires: npm(patternfly-bootstrap-treeview) >= 2.1.10
+Requires: npm(patternfly-bootstrap-treeview) < 2.2.0
```

Comment:
all of these need to be vendored, not packaged independently.

Reply:
The word vendor is not in the readme so not sure what you mean?

Reply:
hah, the correct word is "[bundle](https://github.com/theforeman/foreman-packaging/tree/rpm/develop?tab=readme-ov-file#adding-npm-packages)", which is one of the strategies https://github.com/theforeman/foreman-packaging/blob/rpm/develop/add_npm_package.sh supports (and should have done automatically, given the number of dependencies, weird)